packages/components/src/components/multi-select/multi-select/multi-select-styles.ts

```diff
@@ -0,0 +1,181 @@
+import { getPlaceholderStyles } from '../../../styles/placeholder';
+import { getNoResultsOptionJSSStyles } from '../../../styles/select/option-styles';
+import { getFunctionalComponentRequiredStyles } from '../../common/required/required-styles';
+import { getFunctionalComponentStateMessageStyles } from '../../common/state-message/state-message-styles';
+import { OPTION_HEIGHT } from '../../../styles/select/option-styles';
+
+const inputYPadding = '13px';
+const selectorNativeSelect = 'select,::slotted([slot=select])';
+
+export const getComponentCss = (
+  hasSelection: boolean,
+  direction: SelectDropdownDirectionInternal,
+  isOpen: boolean,
+  isDisabled: boolean,
+  hideLabel: BreakpointCustomizable<boolean>,
+  state: FormState,
+  isWithinForm: boolean,
+  theme: Theme
+): string => {
+  const { primaryColor, contrastMediumColor, contrastHighColor, backgroundColor } = getThemedColors(theme);
+  const { formStateColor, formStateHoverColor } = getThemedFormStateColors(theme, state);
+  const isDirectionDown = direction === 'down';
+
+  return getCss({
+    '@global': addImportantToEachRule({
+      ':host': {
+        display: 'block',
+        position: 'relative',
+        ...hostHiddenStyles,
+      },
+      ...getInputJSSStyles(isDisabled, theme),
+      ...getListJSSStyles(isOpen, direction, theme),
+      ...(isWithinForm && {
+        [selectorNativeSelect]: {
+          opacity: 0,
+          display: 'block',
+          height: '0px',
+        },
+      }),
+    }),
+    root: {
+      position: 'relative',
+    },
+    'input-container': {
+      display: 'flex',
+      color: primaryColor,
+      background: backgroundColor,
+      transition: ['color', 'border-color', 'background-color'].map(getTransition).join(), // for smooth transitions between e.g. disabled states
+      cursor: isDisabled ? 'not-allowed' : 'text',
+      ...hoverMediaQuery({
+        '&:hover:not(.disabled)': {
+          borderColor: isOpen ? primaryColor : formStateHoverColor || primaryColor,
+        },
+      }),
+      '&:focus-within': {
+        borderColor: primaryColor,
+      },
+      border: `${borderWidthBase} solid ${isOpen ? primaryColor : formStateColor || contrastMediumColor}`,
+      borderRadius: borderRadiusSmall,
+      ...(isOpen && {
+        [isDirectionDown ? 'borderBottom' : 'borderTop']: addImportantToRule(`1px solid ${contrastMediumColor}`),
+        [isDirectionDown ? 'borderBottomLeftRadius' : 'borderTopLeftRadius']: 0,
+        [isDirectionDown ? 'borderBottomRightRadius' : 'borderTopRightRadius']: 0,
+      }),
+    },
+    ...getLabelStyles('select', isDisabled, hideLabel, state, theme, undefined, { marginBottom: spacingStaticXSmall }),
+    icon: {
+      padding: `${inputYPadding} 15px`, // Horizontal padding spacingStaticMedium - 1px for visual balance
+      cursor: isDisabled ? 'not-allowed' : 'pointer',
+    },
+    ['reset-icon']: {
```

Comment:
Brackets around keys are only needed for string interpolations or when referencing variables.